clustering/jgroups/extension/src/main/java/org/jboss/as/clustering/jgroups/subsystem/JGroupsModel.java

```diff
@@ -38,8 +38,9 @@
     VERSION_5_0_0(5, 0, 0), // WildFly 11, EAP 7.1
     VERSION_6_0_0(6, 0, 0), // WildFly 12-16, EAP 7.2
     VERSION_7_0_0(7, 0, 0), // WildFly 17
+    VERSION_8_0_0(8, 0, 0), // WildFly 18
```

Comment:
Nope. WildFly 18 still used the model version 7.

Reply:
Wrote this back when we were trying to get this change into WildFly 18 ;) I've updated it to WildFly 20 now, thanks!